libs/utils/src/lib/types/concept.ts

```diff
@@ -0,0 +1,170 @@
+export interface ErrorMessage {
```

Comment:
Foreslår å ha dette under egen lib types og ikke under utils. Alternativet er å lage en lib "common" hvor vi har ting som types og language. Vi kan godt sette opp et lite møte for dette.

Reply:
Prøvde å holde meg til de 4 library types som er forreslått av NX, men ingenting i veien for å ha mer hvis du synes det er mer intuitivt.

Reply:
Det høres lurt ut. Jeg foreslår vi avklarer dette med resten av teamet siden dette kommer til å berøre alle.

Reply:
Vi bør da kanskje flytte types under data-access siden det er tett koblet.

Reply:
@KjerstiSteien @KurtStian @NilsOveTen @valosnah @terjesyl @hegeaal ?

Reply:
Hm, kjenner ikke godt nok til best practice på struktur her, så stoler på deres avgjørelse:) Men høres lurt ut å holde oss til få library-types. Kan refaktorere senere om det blir svært mange typer og vi har behov for å stykke det opp mer.

Reply:
OK. Da lager jeg en bibiliotek for typer.